src/main/java/com/springboot/star_wars_character_popularity/app/model/Character.java

```diff
@@ -28,12 +28,12 @@ public class CharacterModel {
     @Column(name = "votersCount")
```

Comment:
in MySql, columns should be named voters_count, with underscore instead of CamelCase